msteams-platform/concepts/authentication/nested-authentication.md

```diff
@@ -0,0 +1,190 @@
+---
+title: Nested app authentication
+description: Learn about nested authentication in the Microsoft Teams platform.
+ms.date:
+ms.topic: conceptual
+author: v-ypalikila
+ms.author: surbhigupta
+ms.localizationpriority: medium
+---
+
+# Nested app authentication
+
+> [!NOTE]
+> Nested app authentication is available only in [public developer preview](../../resources/dev-preview/developer-preview-intro.md).
+
+Nested app authentication (NAA) is a new authentication protocol for single page applications that are embedded in host environments like Teams, Outlook, and Microsoft 365. It simplifies the authentication process and
+Nested app authentication (NAA) is a new authentication protocol for single page applications that are embedded in host environments like Teams, Outlook, and Microsoft 365. It simplifies the authentication process to facilitate single sign-on (SSO) across applications nested within supported host apps and provides several advantages over the On-Behalf-Of (OBO) flow.
+
+* You only need to use the MSAL.js library and don’t need the getAuthToken function in TeamsJS.
+* You can call services such as Microsoft Graph with an access token from your client code as an SPA. There’s no need for a middle-tier server.
+* You can use incremental and dynamic consent for scopes.
+* You don't need to preauthorize your hosts (For example, Teams, Office) to call your endpoints.
+
+The NAA model supports a primary identity that includes multiple app identities. Microsoft utilizes this framework in Office Add-Ins and Teams Tabs and Personal apps.
+
+The following table outlines the difference between Teams Microsoft Entra ID SSO and Nested App Authentication:
+
+| Steps | Traditional Teams Microsoft Entra ID SSO | Nested App Authentication |
```

Comment:
@lokimeyburg I believe this was pulled from one of your early documents. Could you help validate this is all still true and/or let us know if you have any opinion on it being included in this documentation.

Reply:
@lokimeyburg Request you to please respond to @chrisyanasak' query so that we may proceed.

Reply:
Resolving conversation following Chris' approval on the [channel post](https://teams.microsoft.com/l/message/19:9da4360914d24cf58a6a0c54af6c6211@thread.skype/1709841614006?tenantId=72f988bf-86f1-41af-91ab-2d7cd011db47&groupId=32e3b156-66b2-4135-9aeb-73295a35a55b&parentMessageId=1709841614006&teamName=Teams%20Platform&channelName=Platform%20Doc-Queries&createdTime=1709841614006).